plugins/services/src/js/columns/QuotaOverviewLimitColumn.tsx

```diff
@@ -89,11 +89,70 @@ export function limitRenderer(group: ServiceGroup) {
       <div className="service-table-heading flex-box flex-box-align-vertical-center table-cell-flex-box text-overflow">
         <span className="icon-margin-right">{icon}</span>
         <Trans
-          id={limit}
+          id={limitInfo.limitText}
           render="span"
           className="table-cell-value table-cell-flex-box"
         />
       </div>
     </TextCell>
   );
 }
+
+function itemIsServiceGroup(item: any): item is ServiceGroup {
```

Comment:
that pattern can lead to runtime errors as `itemIsServiceGroup` gladly takes any input, but calls `.quota` on it, which will fail for e.g. all primitive types.